src/Chapter04/Listing04.26.RelyingOnIndentation.cs

```diff
@@ -12,7 +10,7 @@ public static void Main()
             #region INCLUDE
             if (radius >= 0)
                 area = Math.PI * radius * radius;
-                System.Console.WriteLine(
+            Console.WriteLine(
```

Comment:
```suggestion
                Console.WriteLine(
```
Indent the writeline

Reply:
Yup